scrapers/SexLikeReal.yml

```
@@ -6,28 +6,21 @@ sceneByURL:
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
-    common:
-      $info: //div[@class="u-py--four desktop:u-ml--twn-one u-relative"]/div
     scene:
-      Title: //div[@class="u-relative u-z--zero"]//h1/text()
-      Date:
-        selector: $info//time/@datetime
-        postProcess:
-          - parseDate: 2006-01-02
-      Details:
-        selector: $info//div[@class="u-mb--four u-lh--opt u-fs--fo u-fw--medium u-lw"]//text()
-        concat: " "
+      Title: //h1/text()
+      Date: //time/@datetime
+      Details: //div[@id="modalSceneAbout"]//comment()[contains(.,'description')]/following-sibling::div/text()
```

In some cases the description div has anchors so you get partial details (`https://www.sexlikereal.com/scenes/dean-of-admissionary-8736`). Skipping the last `text()` is the only thing needed.
```suggestion
      Details: //div[@id="modalSceneAbout"]//comment()[contains(.,'description')]/following-sibling::div
```